Mathlib/RingTheory/Morita/Basic.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2025 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Yunzhou Xie
+-/
+import Mathlib.Algebra.Category.ModuleCat.Basic
+import Mathlib.Algebra.Category.ModuleCat.ChangeOfRings
+
+/-!
+# Morita equivalence
+
+We say that `R` and `S` are Morita equivalent if the categories of modules over `R` and `S` are
+equivalent. In this file, we prove that Morita equivalence is an equivalence relation and that
+isomorphic rings are Morita equivalent.
+
+# Main definitions
+
+- `IsMoritaEquivalent R S`: a predicate asserting that `R` and `S` are Morita equivalent.
+
+## TODO
+
+- For any ring `R`, `R` and `Matₙ(R)` are Morita equivalent.
+- Morita equivalence in terms of projective generators.
+- Morita equivalence in terms of full idempotents.
+- If `R ≈ S`, then `R` is simple iff `S` is simple.
```

Comment:
Add: Morita equivalence in terms of existence of an invertible bimodule.

Reply:
added, thanks